pkg/rbac/roles.go

```diff
@@ -0,0 +1,75 @@
+package rbac
+
+import (
+	"fmt"
+	"github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	// RolePostfix is the postfix added to the name of the role that will be created to review trigger resources
```

Comment:
nit: your other comment says `Triggers resources` instead of `trigger resources`. I think I prefer `Triggers resources` 🤷‍♂